packages/cbioportal-utils/src/hotspot/CancerHotspotsUtils.ts

```diff
@@ -128,6 +144,9 @@ export function isHotspot(
 export function defaultHotspotFilter(hotspot: Hotspot) {
     const type = hotspot.type.toLowerCase();
     return (
-        type.includes('single') || type.includes('indel') || type.includes('3d')
+        type.includes('single') ||
+        type.includes('indel') ||
+        type.includes('3d') ||
+        type.includes('splice')
```

Comment:
not sure if I should add `splice` into default filter

Reply:
I think it is okay If splice hotspots are considered like any other recurrent hotspot

Reply:
although splice hotspots are not considered like recurrent hotspot, I think we still should add it here. Because this is the default filter, and we use this filter on getting overall hotspots data like `hotspotCount` or `hotspotsByPosition`, we have other filters like `filterSpliceHotspotsByMutations` when we need a particular type of hotspot data